packages/server/graphql/mutations/updateJiraDimensionField.ts

```diff
@@ -77,12 +77,25 @@ const updateJiraDimensionField = {
     if (!auth) {
       return {error: {message: 'Not authenticated with Jira'}}
     }
-    const {accessToken} = auth
-    const manager = new AtlassianServerManager(accessToken)
-    const fields = await manager.getFields(cloudId)
-    const field = fields.find((field) => field.name === fieldName)
-    if (!field) return {error: {message: 'Invalid field name'}}
-    const {id: fieldId, schema} = field
+
+    let selectedField
+    // we have 2 special treatment fields, JIRA_FIELD_COMMENT and JIRA_FIELD_NULL which are handled
+    // differently and can't be found on Jira fields list
+    const customFields = [
+      SprintPokerDefaults.JIRA_FIELD_COMMENT,
+      SprintPokerDefaults.JIRA_FIELD_NULL
+    ]
+    if (customFields.includes(fieldName)) selectedField = {id: fieldName, schema: {type: 'string'}}
```

Comment:
How would you feel about combining the custom fields and regular fields into one array? I.e.
```
const allFields = [...fields, ...customFields]
```
That way we could map through it once and avoid the let & if else logic?

Reply:
@nickoferrall Thank you for checking the PR 🙏 . 
That'd definitely simplify the code. The question is, do we prefer an additional `if` statement and save Jira API call or simpler code with unnecessary Jira API call in case of custom fields usage? 😅

Reply:
True! If you'd prefer to save the Jira hit then it LGTM 🎉